.eslintrc.js

```diff
@@ -56,6 +57,7 @@ module.exports = {
       rules: {
         'no-restricted-properties': 'off',
         'no-restricted-syntax': 'off',
+        'no-console': 'off',
```

Comment:
Are we sure we want to turn this off? I often temporarily add console logs for debugging and would rather not accidentally commit them. I prefer using disable comments to allow console logs where we actually want them committed.

Reply:
This only applies to certain file glob patterns, targeting scripts and test code and other non-app code. I made this change because we do usually want to log in those situations. However, I can also see your point that it's good to have to opt-in to logging, even in scripts and tests. I think I will change this back to requiring explicit opt-in for the reasons you mention.

Reply:
Reverted: 966347aebd